webui_runner/Dockerfile

```diff
@@ -1,31 +1,34 @@
 FROM ubuntu:bionic
 
 RUN apt-get update \
- && apt-get install -y --no-install-recommends \
+	&& apt-get install -y --no-install-recommends \
 	apt-utils build-essential gpg-agent \
 	curl ca-certificates wget software-properties-common \
-    psmisc lsof git cmake nano
+	psmisc lsof git cmake nano
 
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key|apt-key add -
 
-RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main"
+RUN add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
 
 RUN apt-get update \
 	&& apt-get install -y --no-install-recommends \
-	clang-8 lldb-8 lld-8
+	clang-10 lldb-10 lld-10
+#RUN curl -sL https://github.com/CraneStation/wasi-sdk/releases/download/wasi-sdk-6/libclang_rt.builtins-wasm32-wasi-6.0.tar.gz  | tar x -zf - -C /usr/lib/llvm-8/lib/clang/8.0.1
```

Comment:
Maybe commented out code should be removed instead?

Reply:
Hi ... yes! Sorry. Should have double checked for comments.